civis/civis.py

```diff
@@ -13,6 +13,45 @@
 
 
 def find(object_list, filter_func=None, **kwargs):
+    """Return the objects from ``object_list`` that satisfy the filters.
+
+    Parameters
+    ----------
+    object_list : iterable
+        An iterable of arbitrary objects, particularly dicts and objects with
+        attributes that can be targeted by the filters in ``kwargs``.
+    filter_func : callable, optional
+        A one-argument function. If specified, ``kwargs`` are ignored.
+        An ``object`` from the input iterable is kept in the returned list
+        if and only if ``bool(filter_func(object))`` is ``True``.
+    **kwargs
+        Key-value pairs for more fine-grained filtering; they cannot be used
+        in conjunction with ``filter_func``. All keys must be strings.
+        An ``object`` from the input iterable is kept in the returned
+        list, unless one of the following conditions is met:
+        - ``key`` is not an attribute of ``object``
+        - ``value`` is a one-argument function and
+          ``bool(value(object[key]))`` is ``False``
+        - ``value`` is ``False``
+        - ``object[key]`` is not equal to ``value``
+
+    Returns
+    -------
+    list
+
+    Examples
+    --------
+    >>> import civis
+    >>> client = civis.APIClient()
+    >>> # creds is a list of dicts representing credentials
```

Comment:
It's a list of `Response` objects. That's an important distinction, because this example won't work on a list of dictionaries.